src/tests/stdio/fgets_test.c

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file
+ * @brief
+ *
+ * @author  Nastya Nizharadze
+ * @date    08.07.2019
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("stdio/fgets test");
+
+/* File's content is
+* qwer\n
+* asdf\n
+* \n
+* zxcv
+*/
+#define TEST_FILE_NAME       "/test_fgets.txt"
+#define TEST_FILE_NAME_EMPTY "/test_fgets_empty.txt"
+
+#define STR_SIZE_TEST (6 * sizeof(char))
```

Comment:
What is 6. A magic number?

Reply:
in file "test_fgets.txt" all lines is shorter or equals 4. +1 for \n and +1 because fgets manual says that fgets() reads in at most one less characters than the input size from file.